src/dataflow/src/source/mod.rs

```diff
@@ -1908,18 +1508,6 @@ fn maybe_emit_timestamp_bindings(
     // Emit required changes downstream.
     let to_emit = changes
         .into_iter()
-        .filter(|((source_ts, assigned_ts), _diff)| {
```

Comment:
@petrosagg Did you remove this because it was panicking?

Reply:
Ah, first I removed the ConsistencyInfo struct and then removed this because I thought it was impossible to get into this situation since we make sure each (partition, source_instance) combination for a given source ends up on the same worker. Did I break something? I can add it back